000-orm-in-databuilder.md

```diff
@@ -0,0 +1,97 @@
+- Feature Name: Support ORM in databuilder
+- Start Date: 2020-10-26
+- RFC PR: [amundsen-io/rfcs#0010](https://github.com/amundsen-io/rfcs/pull/10)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Support ORM in databuilder
+
+## Summary
+
+Currently, Amundsen has a great support on graph DB as backend store. However, in order to use MySQL as the metadata 
+store in Amundsen, we have to support ORM for relational DB first in databuilder. This RFC is mainly about the relational DB schema plan, model translator, and the corresponding 
+change in data transformer, loader and publisher. 
+## Motivation
+
+Relational DBs, like MySQL, are well deployed and widely used in most workplaces. Using the well-established DB infrastructure would save the cost(e.g., new DB deployment, backup, monitoring, proxy) 
+for potential Amundsen users. Hence, it would be helpful if Amundsen can support MySQL as metadata store. The expected result is user is able to publish extracted metadata to MySQL via 
+data loader, transformer, and elastic search publisher would work with ORM as well.
+
+## Guide-level Explanation (aka Product Details)
+
+There would be new transformers, loader, publisher for ORM/MySQL use. For all current graph db users, they do not have to make any change. 
+For new users who want to use MySQL as the backend store, they can still use the existing extractors and then pass new data transformer, loader to ETL task and the new publisher to job executor.
+
+
+## UI/UX-level Explanation
+
+N/A
+## Reference-level Explanation (aka Technical Details)
+
+- Still use the existing models but add model translators to support ORM schema definition and map node/relation data to the record that can be better consumed by relational DBs. 
```

Comment:
FYI https://github.com/amundsen-io/amundsendatabuilder/pull/380 which makes databuilder record more generic is just merged